src/Console/Command/ObfuscateData.php

```diff
@@ -0,0 +1,225 @@
+use Ohanzee\DB;
+use Faker;
+
+class ObfuscateData extends Command
+{
+    private $postRepository;
+    private $contactRepository;
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'db:obfuscatedata';
+
+    /**
+     * The console command signature.
+     *
+     * @var string
+     */
+    protected $signature = 'db:obfuscatedata';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Obfuscates selected PII and removes various config keys from database';
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    public function handle()
+    {
+        $this->db = service('kohana.db');
+        $this->contactRepository = service('repository.contact');
+        $this->postRepository = service('repository.post');
+        $this->configRepository = service('repository.config');
+        $this->userRepository = service('repository.user');
+
+        if ($this->isTestDeployment() || $this->isStagingDeployment()) {
+            //confirm acknowledgement
+            $this->info("This script will wipe data from the current deployment.\n\n");
+
+            if (!$this->confirm('Do you acknowledge that this data will still contain sensitive data?')) {
+                $this->info("Request canceled.\n");
+                exit;
+            }
+
+            // Do overwriting
+            $this->db->begin();
+            $this->overwriteContacts();
+            $this->overwritePostAuthors();
+            $this->removeDataProviderValues();
+            $this->overwriteSiteConfig();
+            $this->deleteUsers();
+            $this->addAdminUser();
+            $this->db->commit();
+        } else {
+            $this->info("This script will only run on test or staging deployments.\n\n");
+            exit;
+        }
+    }
+
+    private function isTestDeployment()
+    {
+        return (strtolower(getenv('APP_ENV')) == 'test');
+    }
+
+    private function isStagingDeployment()
+    {
+        return (strtolower(getenv('APP_ENV')) == 'staging');
+    }
+
+    /* optional implementations...
+    private function randomizeLongitudeLatitude($boundaries)
+    {
+
+    }
+    */
+
+    /* TODO: would this be useful?
+    private function scrambleLongitudeLatitude($distanceFromOrigin = null)
+    {
+
+    }
+    */
+
+    private function overwriteContacts()
+    {
+        $this->info("Overwriting contact data...");
+        $faker = Faker\Factory::create();
+        $query = DB::select('contacts.id', 'contacts.type', 'contacts.contact')
+            ->distinct(true)
+            ->from('contacts');
+        $results = $query->execute($this->db);
+        $overwrittenCount = 0;
+        $resultsCount = count($results);
+        foreach ($results as $row) {
+            $contactEntity = $this->contactRepository->getEntity($row);
+            $fakePhone = $faker->e164PhoneNumber;
```

Comment:
Not sure if you just haven't got to this yet, but contacts can be emails or twitter handles too